source/win-main/MainSidebar.vue

```diff
@@ -18,6 +18,7 @@
         v-for="(entry, idx) of tableOfContents"
         v-bind:key="idx"
         class="toc-entry-container"
+        v-bind:class="{'toc-active': tocIsActive(entry.line, idx)}"
```

Comment:
Maybe rename to toc-item-active to make sure that not the full toc is active ...?

Reply:
Done.

Btw, I renamed it to `toc-entry-active`, or is there a difference between ToC "item" and "entry"?